packages/framework-provider-kubernetes-infrastructure/src/infrastructure/dapr-manager.ts

```diff
@@ -97,8 +97,8 @@ export class DaprManager {
    * remove dapr services from your cluster
    */
   public async deleteDaprService(): Promise<void> {
-    const { stderr } = await this.helmManager.exec(`uninstall dapr -n ${this.namespace}`)
-    if (stderr) throw new Error(stderr)
+    const { stdout, stderr } = await this.helmManager.exec(`uninstall dapr -n ${this.namespace}`)
+    if (!stdout && stderr) throw new Error(stderr)
```

Comment:
Warnings are returned inside `stderr`, so I need to check both `stdout` and `stderr`